components/ai_chat/resources/page/components/welcome_guide/style.module.scss

```diff
@@ -3,60 +3,177 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // you can obtain one at https://mozilla.org/MPL/2.0/.
 
-.box {
+.wrapper {
+  --leo-button-color: var(--leo-color-text-tertiary);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
   padding: var(--leo-spacing-xl) var(--leo-spacing-xl) 0;
-  overflow: auto;
-
-  p {
-    margin: 0;
-  }
+  gap: var(--leo-spacing-4xl);
+  container-type: inline-size;
 }
 
 .header {
-  padding: var(--leo-spacing-2xl);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: var(--leo-spacing-l);
 }
 
 .title {
   font: var(--leo-font-heading-h1);
-  margin: 0;
-  margin-bottom: var(--leo-spacing-m);
+  color: var(--leo-color-text-primary);
+  text-align: center;
 }
 
 .subtitle {
   font: var(--leo-font-heading-h2);
-  color: var(--leo-color-text-tertiary);
-  margin: var(--leo-spacing-m) 0;
+  color: var(--leo-color-text-secondary);
+  text-align: center;
+}
+
+.subtitlePurple {
+  color: var(--leo-color-private-window-40);
```

Comment:
hmmm this seems wrong - is this what the designer used?

cc @aguscruiz I think this should probably be using the primary color, otherwise it isn't going to match the theme